tests/test_sockets.py

```diff
@@ -735,12 +735,17 @@ async def test_bind_link_local(self) -> None:
     sys.platform == "win32", reason="UNIX sockets are not available on Windows"
 )
 class TestUNIXStream:
-    @pytest.fixture
-    def socket_path(self) -> Generator[Path, None, None]:
+    @pytest.fixture(params=["path", "abstract"])
```

Comment:
The latter parameter needs a `skipif` marker when not on Linux, as this is a Linux specific feature.

Reply:
Fixed